src/riak_kv_vnode.erl

```diff
@@ -1013,41 +1079,47 @@ handle_command({fold_indexes, FoldIndexFun, Acc}, Sender, State=#state{mod=Mod,
     end;
 
 
-handle_command({rebuild_complete, store}, _Sender, State) ->
+handle_command({rebuild_complete, store, ST}, _Sender, State) ->
```

Comment:
This is an exported funtion, used by riak_core_vnode... I trust that all places that it needs 3 instead of 2 tuple are checked.